src/Glpi/Debug/Profiler.php

```diff
@@ -51,6 +51,8 @@ final class Profiler
     public const CATEGORY_DB = 'db';
     public const CATEGORY_TWIG = 'twig';
     public const CATEGORY_SEARCH = 'search';
+    public const CATEGORY_CUSTOMOBJECTS = 'customobjects';
+    public const CATEGORY_SYMFONY = 'symfony';
```

Comment:
This is not really related to Symfony itself. Indeed, everything in this section is actually related to the GLPI framework initialization / bootstrapping.

```suggestion
    public const CATEGORY_GLPI_BOOT = 'glpi-boot';
```